### ISSUE TITLE
[BETA-BUG] App is set to system theme, it shows Light (text), but set is Dark theme (that is default in system)

### ISSUE THREAD

**Issue by inson1** [Darkempire78/OpenCalc#353]
![Screenshot_2023-11-05-22-04-22-811_com.darkempire78.opencalculator.jpg](https://github.com/Darkempire78/OpenCalc/assets/75314629/0959f388-37bb-4e7f-b14c-7f0d9d5bee94)

Device: Xiaomi Redmi note 8 Pro
OS: Android 11 MIUI 12.5.7
Version: 3.0.0 beta 4
Language: English

**Comment by Darkempire78** [comment]
It seems to be a duplicate of #352

**Comment by inson1** [comment]
@Darkempire78 No, system theme was Dark, not Light. In #352 just the Light theme works like lighter dark theme
